### ISSUE TITLE
Getting stuck in forced cameras when scrubbing

### ISSUE THREAD

**Issue by BlacRyu** [XertroV/tm-ghosts-plus-plus#16]
Camera triggers only seem to apply if you view the part of the run where the trigger occurs. If you scrub past a trigger, either forward or back, it will not activate and you will stay in whatever camera you were in before.

For instance, if you are watching a run that enters a loop and begins a forced camera 3, then you scrub back to the start, you will still be stuck in camera 3 until you get to the end of the loop where the car hits the reset camera trigger.

Likewise if you start watching a run then scrub to the middle of a loop, you will not be in cam 3 even if you normally would be when watching the ghost.

It would be nice if Ghost++ could track which sections of the run should be in which camera, and trigger camera changes when scrubbing from one part to another.

**Comment by XertroV** [comment]
yeah, this is a limitation of the game engine. While it might be *technically* possible to do that, screwing with mediatracker is frowned upon so it's usually considered off limits. I can ask in the op discord, though.

**Comment by BlacRyu** [comment]
If modifying the camera directly isn't possible, perhaps you could make it so that when clicking on the timeline, it first jumps to the frame where the camera last changed before jumping to where the user clicked. Maybe a bit jank, but the game would still be handling the camera changes for you in that case. 

One problem with that is that there's probably rarely a camera reset before the first forced camera, so you can't undo the first camera change unless you jump to some other random frame that resets it.

Anyways it's not a huge issue, just a bit annoying. Guess we'll just have to live with it for now if it can't be fixed. Thanks for looking into it.
